assets/home/sections/sensei-pro-ad.scss

```diff
@@ -115,29 +115,35 @@
 		}
 
 		&__card {
-			display:flex;
-			justify-content: center;
-			align-items:center;
 			background-color: $white;
-			flex-direction: column;
-			padding: 24px;
 			margin-top: 48px;
 
-			&--image, &--cta {
+			&--image {
 				width: 100%;
 			}
 
-			&--cta {
-				max-width: 300px;
+			&--quote {
+				max-width: 335px;
```

Comment:
My screen doesn't match your screenshot with this. The stars are forced down to the next line:
<img width="1270" alt="Screenshot 2022-10-26 at 9 35 04 PM" src="https://user-images.githubusercontent.com/68693/198132068-542b6899-fad3-404b-838a-93e711bbeab4.png">

Reply:
Hey there! Thanks for the review! This should be fixed after d1155ea33dc6bd6a8d6007ea28a248259c9823d2